package.json

```diff
@@ -13,16 +13,17 @@
     "lint": "stylelint **/*.scss && eslint . --ext .js,.jsx,.ts,.tsx",
     "build": "yarn webpack --config webpack.config.js",
     "build:prod": "NODE_ENV=production yarn webpack --mode production --config webpack.config.js",
-    "test": "jest && jest -c jest.routes.config.js",
+    "test": "npx if-env CI=true && exit 0 || yarn test:unit",
```

Comment:
The above will remove the running of the route tests from the pipeline. The routes bit will need adding in to the above (or we could revert back the above and raise as a separate ticket to cover this bit)

Reply:
Yeah - have fixed this now :)